website/source/docs/job-specification/constraint.html.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: "docs"
+page_title: "constraint Stanza - Job Specification"
+sidebar_current: "docs-job-specification-constraint"
+description: |-
+  The "constraint" stanza specifies non-resource requirements. Constraints may
```

Comment:
The "constraint" stanza specifies non-resource requirements -> The "constraint" stance allows restricting the set of eligible nodes.

It isn't necessarily true that it is non-resource related